web_widget_mail_send_odoo/static/src/js/web_widget_mail_send_odoo.js

```diff
@@ -0,0 +1,91 @@
+/* © 2015 initOS GmbH (<http://www.initos.com>)
+*  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+*/
+
+(function() {
```

Comment:
This runs as soon as the javascript code is loaded, which can lead to timing problems. Use the standard pattern to define a function in the openerp namespace with the addon's name, getting `instance` as parameter: https://github.com/OCA/web/blob/8.0/support_branding/static/src/js/support_branding.js#L16